README.md

```diff
@@ -88,11 +88,13 @@ config :nerves_firmware_ssh,
   ]
 ```
 
-By itself, this does not allow you to log into your device using `ssh`. This is
-only for sending firmware updates to the device. (The `ssh` protocol is really
-cool and lets you do more than just connect to shells.) If you'd like to connect
-to the IEx prompt, see the [:ssh_console_port](#ssh_console_port) configuration
-option.
+These keys also let you log into your Nerves device and get an IEx prompt.  See
+the [:ssh_console_port](#ssh_console_port) configuration option.
+
+There has been some confusion with `ssh` support. IEx prompt access and firmware
```

Comment:
I don’t think it’s necessary to point out that’s there has been confusion in the past, since you’re now clarifying the docs to remove the confusion.

Reply:
I agree now that I read it again. Fixed. Thanks.